settings.gradle

```diff
@@ -19,4 +22,28 @@ buildCache {
 		enabled = true
 		removeUnusedEntriesAfterDays = 7
 	}
+	Properties localProperties = new Properties()
+	File file = new File("local.properties")
+	if (file.exists()) {
+		localProperties.load(file.newDataInputStream())
+	}
+	String streamAWSRegion = localProperties.get("buildCache.AWSRegion") ?: System.getenv("BUILD_CACHE_AWS_REGION") ?: ""
+	String streamAWSBucket = localProperties.get("buildCache.AWSBucket") ?: System.getenv("BUILD_CACHE_AWS_BUCKET") ?: ""
+	String streamAWSAccessKeyId = localProperties.get("buildCache.AWSAccessKeyId") ?: System.getenv("BUILD_CACHE_AWS_ACCESS_KEY_ID") ?: ""
+	String streamAWSSecretKey = localProperties.get("buildCache.AWSSecretKey") ?: System.getenv("BUILD_CACHE_AWS_SECRET_KEY") ?: ""
+	if (!streamAWSRegion.isBlank() &&
+			!streamAWSBucket.isBlank() &&
+			!streamAWSAccessKeyId.isBlank() &&
+			!streamAWSSecretKey.isBlank()
```

Comment:
Perhaps we could add an extra local property like `buildCache.disable` that devs can use to quickly disable build cache (for example, when working offline, or on a poor connection where they don't wish to use it)

Reply:
I have added an extra property, then you don't need to comment/uncomment properties on your side